libdrgn/drgn.h.in

```diff
@@ -2622,6 +2640,14 @@ enum drgn_symbol_kind drgn_symbol_kind(struct drgn_symbol *sym);
 /** Return whether two symbols are identical. */
 bool drgn_symbol_eq(struct drgn_symbol *a, struct drgn_symbol *b);
 
+/**
+ * Destroy each @ref drgn_symbol in @syms, and free the array.
+ *
+ * This will ignore any @c NULL entry in the array, allowing you to take
+ * ownership of any symbol from the array prior to freeing the rest.
```

Comment:
You should probably mention that symbols taken in this way must be destroyed with `drgn_symbol_destroy()`.